apps/desktop/src/lib/vbranches/types.ts

```diff
@@ -476,6 +457,22 @@ export class PatchSeries {
 	get integrated() {
 		return this.patches.length > 0 && this.patches.length === this.integratedCommits.length;
 	}
+
+	get ancestorMostConflictedCommit(): DetailedCommit | undefined {
+		const commits = [...this.upstreamPatches, ...this.patches].sort((a, b) => {
+			if (a.createdAt > b.createdAt) {
```

Comment:
I wonder how this works with commits that were created in the middle of the series.

I believe this is only used to check if this is the 'first' conflicted from the bottom, so in order to avoid having to sort them, I think we can assume that there are no `upstreamPatches` that are also conflicted.

Maybe we can update that horrible name I gave this getter and make it even worse: `ancestorMostConflictedLocalCommit`

Reply:
Haha okay yeah I cleaned this up a bit. In terms of naming, I thought about it for a minute, but seeing as want to avoid "top"/"bottom" I coudln't think of anything better :joy:

Reply:
Haha fair enough